src/test/java/com/google/firebase/auth/GetUsersIT.java

```diff
@@ -0,0 +1,152 @@
+/*
```

Comment:
junit4 doesn't allow nested tests (and therefore doesn't allow nested setup/teardown functions.) The recommendation is to just split into a separate file, which is what I've done here.